### PR TITLE
Make cube2eq batched

### PR DESCRIPTION
**Pull request by Skylion007**
## Motivation and Context

It improves the speed of the Cube2Equirectangular ObservationTransformer by fulling batching the input. Improve speed by nearly 10%, the overhead of the transformation is now minimal. Also fixes a bug where it couldn't get properly send input to more than two target sensors.

Finally, it reduces the number of tests run in test_baseline_trainers so that the massive testing time regression is now fixed.

## How Has This Been Tested
Locally with PyTest and manually looking at the output + benchmarking. Increases FPS from 100-120 locally.

## Types of changes

-  refactoring 

## Checklist

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: habitat_baselines/common/obs_transformers.py [hunk 1]
```diff
@@ -418,66 +436,42 @@ def get_grid2(self):
...
+            batch_size // 6, 6, -1, self.equ_h, self.equ_w
+        )  # batch_size//6, 6, ch, self.equ_h, self.equ_w
+        sampled_image_masked = sampled_image * (
+            self.mask.float()
+            .view(-1, 6, 1, self.equ_h, self.equ_w)
+            .expand(1, 6, ch, -1, -1)
+        )  # batch_size//6, 6, ch, self.equ_h, self.equ_w
+        return torch.sum(
+            sampled_image_masked, dim=1
+        )  # batch_size//6, ch, self.equ_h, self.equ_w
```
Comment (mathfac) [inline habitat_baselines/common/obs_transformers.py:462]:
```suggestion
        )  # batch_size // 6, ch, self.equ_h, self.equ_w
```

File: habitat_baselines/common/obs_transformers.py [hunk 2]
```diff
@@ -418,66 +436,42 @@ def get_grid2(self):
...
+            align_corners=False,
+            padding_mode="border",
+        ).view(
+            batch_size // 6, 6, -1, self.equ_h, self.equ_w
+        )  # batch_size//6, 6, ch, self.equ_h, self.equ_w
+        sampled_image_masked = sampled_image * (
+            self.mask.float()
+            .view(-1, 6, 1, self.equ_h, self.equ_w)
+            .expand(1, 6, ch, -1, -1)
+        )  # batch_size//6, 6, ch, self.equ_h, self.equ_w
```
Comment (mathfac) [inline habitat_baselines/common/obs_transformers.py:459]:
```suggestion
        )  # batch_size // 6, 6, ch, self.equ_h, self.equ_w
```

**Comment by Skylion007** [comment]
I need to merge this with #485

**Comment by Skylion007** [comment]
Closing in favor of #485

**Pull request closed without merging**